Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM python:3.12.3 as base
+FROM python:3.12.3 AS base
 RUN curl -sSL https://install.python-poetry.org | python3 -
 ENV PATH=$PATH:/root/.local/bin/
-COPY pyproject.toml poetry.toml /opt/todoapp/
+COPY pyproject.toml poetry.toml .env.test /opt/todoapp/ 
```

Comment:
Interesting, _almost_ everyone moves this COPY for `.env.test` into the `test` stage. I don't have a preference either way; this adds a tiny file into the dev/prod builds unnecessarily, moving it later adds an extra instruction. Simply interesting that it's a different approach to most!